MAINTAINERS-example

```diff
@@ -1,33 +1,23 @@
-# This project, topsikrit, has top-secret awesome sauce.
-#
-# Btw, note that this comment was human-edited and will be preserved
-# even if you add/remove maintainers using the gem. Nifty!
-
-maintainers: {
-    version: 1
-    issues: "https://github.com/csprojects/topsikrit/issues"
-    # This is a very interesting comment. It will also be preserved.
-    roles: {
-        core: {
-            "ada": {},
-            "turing": {},
-        },
-        subsystem: {
-            "gracehopper": { subsystem: "ENIAC" },
-        },
+{
+  "version": 1,
+  "maintained": false,
+  "people": {
+    "gracehopper": {
```

Comment:
A bit of a nitpick, but using arbitrary data as map keys (as opposed to a fixed set of keys) seems to cause more trouble than it's worth. Probably can just make this an array of people. It's also not clear what the key here means. It seems to correspond to github name, in which case it seems redundant?